### PR TITLE
Add usage of ConfigMap to README

### PR DESCRIPTION
**Pull request by leszko**

### CODE REVIEW

File: hazelcast-enterprise-kubernetes/README.md [hunk 1]
```diff
@@ -179,14 +187,85 @@ spec:
...
+    spec:
+      containers:
+      - name: hazelcast-ee-node
+        image: hazelcast/hazelcast-enterprise-kubernetes:3.10
+        ports:
+        - containerPort: 5701
+        livenessProbe:
+          exec:
+            command:
+            - "./liveness.sh"
```
Comment (googlielmo) [inline hazelcast-enterprise-kubernetes/README.md]: we'll have to update this to httpGet :-)

File: hazelcast-kubernetes/README.md [hunk 2]
```diff
@@ -171,14 +179,78 @@ spec:
...
+    spec:
+      containers:
+      - name: hazelcast-node
+        image: hazelcast/hazelcast-kubernetes:3.10
+        ports:
+        - containerPort: 5701
+        livenessProbe:
+          exec:
+            command:
+            - "./liveness.sh"
```
Comment (googlielmo) [inline hazelcast-kubernetes/README.md]: ditto

File: hazelcast-enterprise-kubernetes/README.md (hunk 1 above)
Comment (leszko) [reply hazelcast-enterprise-kubernetes/README.md]: done

File: hazelcast-kubernetes/README.md (hunk 2 above)
Comment (leszko) [reply hazelcast-kubernetes/README.md]: done

**Pull request merged**